### PR TITLE
Prepared for release a new 1.0.0 version

### PR DESCRIPTION
**Pull request by egor-bryzgalov**
Merged `releases/1.x` branch into master.
Changed package version (increased major version, get rid of preview postfix.

Created a backup branch: [releases/0.x](https://github.com/microsoft/azure-pipelines-tool-lib/tree/releases/0.x)

### CODE REVIEW

**Comment by egor-bryzgalov** [comment]
To do: update `task-lib` package version after the `3.x.x` version released.

**Review by anatolybolshakov — APPROVED** [review]

**Review by DaniilShmelev — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-tool-lib",
-  "version": "0.13.2",
+  "version": "1.0.0",
```
Comment (damccorm) [inline package.json:3]: I think this needs to be `1.0.1` to be higher than the current published `1.0.0-preview.0`

**Review by damccorm — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (egor-bryzgalov) [reply package.json]: Thanks! Fixed it

**Comment by egor-bryzgalov** [comment]
@damccorm Could you please publish this? Thank you!

**Comment by damccorm** [comment]
Yep, published!
________________________________

**Pull request merged**